_episodes/02-keras.md

```diff
@@ -432,7 +432,7 @@ The model summary here can show you some information about the neural network we
 > > {:.language-python}
 > >
 > > ~~~
-> Model: "functional_1"
+> Model: "model_1"
```

Comment:
this one probably depends on tensorflow versions, in my environment (tensorflow=2.4.1) I get the name `functional_3`. Not too important I guess, should we make a note on it? We could also define the name manually.

Reply:
A note or call-out box makes sense from my point of view. Especially considering self-paced learners. @wmotion could you include that?

Reply:
I have added one line mentioning that name differences are not important.
https://github.com/wmotion/deep-learning-intro/blob/4057dc0b9294a13ebe73022618d7ca11de3c5cd2/_episodes/02-keras.md?plain=1#L454